db/migrate/20180517164500_create_disability_compensation_submissions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDisabilityCompensationSubmissions < ActiveRecord::Migration
+  def change
+    create_table :disability_compensation_submissions do |t|
+      t.uuid :user_uuid, null: false
+      t.string :form_id, null: false
```

Comment:
This ID is for the type of form, correct? Not a unique ID for the form submission? I prefer `form_type` in that case, but I know this deviates from a lot of other places where we use `form_id`...